types/deployment_stage_interface.ts

```diff
@@ -1,3 +1,26 @@
+import BigNumber from 'bignumber.js';
+
 export interface DeploymentStageInterface {
   deploy(web3: any): Promise<any>;
+}
+
+interface NetworkConstants {
+	production: any;
```

Comment:
Think these should be numbers vs any type

Reply:
Disagree cause there might be other kinds of constants we want to tweak in the future that aren't limited to numbers. Could be addresses and what not.